src/transformers/models/auto/modeling_auto.py

```diff
@@ -424,6 +425,7 @@
         ("mega", "MegaForMaskedLM"),
         ("megatron-bert", "MegatronBertForCausalLM"),
         ("mobilebert", "MobileBertForMaskedLM"),
+        ("moonshine", "MoonshineForConditionalGeneration"),
```

Comment:
Maybe also worth adding to `MODEL_FOR_SPEECH_SEQ_2_SEQ_MAPPING_NAMES` ? (whisper is there)

Reply:
Added! thanks 😊